packages/frontend/src/features/metricsCatalog/components/visualization/MetricsVisualization.tsx

```diff
@@ -157,91 +198,102 @@ const MetricsVisualization: FC<Props> = ({ data, comparison }) => {
                 </Button>
             </Group>
 
-            <ResponsiveContainer width="100%" height="100%">
-                <LineChart
-                    data={activeData}
-                    margin={{ top: 40, right: 40, bottom: 40, left: 40 }}
-                    onMouseDown={handleMouseDown}
-                    onMouseMove={handleMouseMove}
-                    onMouseUp={handleMouseUp}
-                >
-                    <CartesianGrid
-                        horizontal
-                        vertical={false}
-                        stroke={colors.gray[2]}
-                        strokeDasharray="4 3"
-                    />
+            {showEmptyState && <MetricsVisualizationEmptyState />}
```

Comment:
empty state is now in metrics visualization, as well as the date filters' components.